index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style/style.css">
+    <title>Kemboi Portfolio</title>
+</head>
+
+<body>
+    <header>
+
+        <div>
+            <section class="logo"><a href="#">Kemboi</a></section>
+            <section class="menu"><img src="/images/union.svg"></section>
+        </div>
+
+        </div>
+
+    </header>
+    <main>
+        <div>
+        </div>
```

Comment:
- [ ] Please make sure the you remove this empty element to make your code clean. :pray:

Reply:
I've removed the empty elements, my code is cleaner. thanks